elyra/pipeline/kfp/processor_kfp.py

```diff
@@ -373,18 +369,12 @@ def process(self, pipeline):
         )
 
     def export(self, pipeline, pipeline_export_format, pipeline_export_path, overwrite):
-        if pipeline_export_format not in ["yaml", "py"]:
+        if pipeline_export_format not in ["yaml"]:
```

Comment:
I had that question too. I followed what was in `processor_airflow.py`, but now I'm thinking I may just change both.

Reply:
I think this should remain an array because `export` is in the "contract" for runtime processors and some implementations may choose to support multiple export formats.  It just so happens that both of our implementations (now) do not, but I believe this should remain a `List`.